src/scraper/scrape.test.ts

```diff
@@ -51,4 +51,41 @@ describe("getAccountTransactions", () => {
       "END_SCRAPING",
     );
   });
+
+  describe("OTP functionality", () => {
```

Comment:
Since we don't have real credentials and real telegram in the test, this test is no testing anything and needs to be removed. if as a result of the other comments you will add a new otp file, unit tests need to be done for the logic